src/.vuepress/styles/index.scss

```diff
@@ -84,3 +87,12 @@ html.dark {
 div.theme-default-content:not(.custom), footer.page-meta:not(.custom), nav.page-nav:not(.custom) {
   max-width: 1800px;
 }
+
```

Comment:
how does this change relate to "better" dark mode?

Reply:
line 90?

Reply:
formatting and i dont see an issue including that when modifying the file

Reply:
> GH doesn't allow you to select multiple lines to comment on

I recently learned it does actually, try holding shift while clicking on any additional lines